### ISSUE TITLE
[win10] Crash after running, exception: STATUS_ACCESS_VIOLATION

### ISSUE THREAD

**Issue by DingpingZhang** [tauri-apps/tauri#2188]
I tried to execute `yarn tauri dev` command, or `yarn tauri build` command and then run app_name.exe from `target/release`, both encountered the following issue: the window appears 2 seconds after the crash exit, the window appears during the white screen, no normal web page content is displayed.

```
$ yarn tauri dev 
yarn run v1.22.10
$ tauri dev
 app:spawn Running "C:\Users\xxx\source\repos\tauri_demo\node_modules\@tauri-apps\cli\bin\tauri-cli.exe tauri dev" +0ms

   Compiling app v0.1.0 (C:\Users\xxx\source\repos\tauri_demo\src-tauri)
    Finished dev [unoptimized + debuginfo] target(s) in 10.63s
     Running `target\debug\app.exe`
error: process didn't exit successfully: `target\debug\app.exe` (exit code: 0xc0000005, STATUS_ACCESS_VIOLATION)
```

Environment info:
```
Operating System - Windows, version 10.0.19042 X64
Webview2 - 91.0.864.64

Node.js environment
  Node.js - 14.17.3
  @tauri-apps/cli - 1.0.0-beta.5
  @tauri-apps/api - 1.0.0-beta.4

Global packages
  npm - 6.14.13
  yarn - 1.22.10

Rust environment
  rustc - 1.53.0
  cargo - 1.53.0

App directory structure
/node_modules
/src-tauri
/src-web

App
  tauri.rs - 1.0.0-beta.4
  build-type - bundle
  CSP - default-src blob: data: filesystem: ws: wss: http: https: tauri: 'unsafe-eval' 'unsafe-inline' 'self' img-src: 'self'
  distDir - ../src-web/index.html
  devPath - ../src-web/index.html
```

From the event viewer log of the win system, we can see that the exception from user32.dll.

![image](https://user-images.githubusercontent.com/8541016/125038771-a7105800-e04a-11eb-9b2c-8c861e74aa63.png)

**Comment by amrbashir** [comment]
Is your repo public? I need to see what apis are used.

**Comment by DingpingZhang** [comment]
@amrbashir I'm sorry to tell you: this crash issue cannot reproduce after I restarted my computer, but it was indeed 100% reproduce before I restarted the win system. I don't know what happened.
I didn't apply any api, I just ran the helloworld example. if I can reproduce it next time, I'll let you know.

By the way, I sometimes find that of course when I run any command, the following result appears and I have to wait a while before it works, but rustlang environment was installed.

```
$ yarn tauri dev (or yarn tauri init, yarn tauri info, yarn tauri build)
yarn run v1.22.4
$ C:\Users\xxx\source\github\tauri_demo\node_modules\.bin\tauri dev
Downloading Rust CLI...
Done in 0.53s.
```
